google/cloud/support/doc/override-endpoint.dox

```diff
@@ -0,0 +1,43 @@
+/*!
+@page support-override-endpoint How to Override the Default Endpoint
+
+In some cases, you may need to override the default endpoint used by the client
+library. Use the
+[`EndpointOption`](@ref google::cloud::EndpointOption) when initializing the
```

Comment:
Probably my fault, this should be `[\c EndpointOption]` Doxygen gets confused about backticks in links.